### ISSUE TITLE
DRAW 0.0.1 is the latest version available when running "flutter packages get"

### ISSUE THREAD

**Issue by PiwwowPants** [draw-dev/DRAW#37]
When adding draw as a pubspec dependency (using Flutter), the latest version found is 0.0.1.
To get 0.0.3, it must be cloned from GitHub (or just getting the pub archive) and included as a local dependency

**Comment by bkonyi** [comment]
Would you mind sharing your pubspec with me? I haven't tried adding it as a dependency for Flutter before, but this did work with my [moderation demo](https://github.com/bkonyi/Toxicity-Moderator/blob/master/pubspec.yaml) from DartConf 2018.

**Comment by PiwwowPants** [comment]
Sure. It's just straight out of the example from the first flutter generation

```
name: example
description: A new Flutter project.

dependencies:
  flutter:
    sdk: flutter
  draw:
    ^0.0.3

  cupertino_icons: ^0.1.0

dev_dependencies:
  flutter_test:
    sdk: flutter

flutter:

  uses-material-design: true
```

**Comment by bkonyi** [comment]
Thanks for posting that. Just curious, what platform is this on? I'll look into all of this today.

**Comment by PiwwowPants** [comment]
Windows 10 1803 17112.1.
Targeting Android with Gradle 3.0.1 for building (doesn't get that far anyway)

**Comment by bkonyi** [comment]
That was odd. I'm not sure why that didn't work, but I've gone ahead and published version 0.1.0 with `package:quiver` rolled to the same version as `flutter_test`. I've successfully pulled DRAW using `flutter package upgrade` so I think this is fixed now. Please let me know if you're still having issues!
